examples/vnf-blueprint.yaml

```diff
@@ -0,0 +1,174 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+description: >
+  This runs a trivial deployment in kubernetes containing single-container pod with virtual ethernet interfaces exposed.
+  It should be run as internal component of one of "service_chain" blueprints.
+  Expects that you have first installed https://github.com/cloudify-examples/simple-kubernetes-blueprint/tree/4.0.1.
+  Also install the wagon with the command: "cfy plugins upload PATH_TO_WAGON".
+
+imports:
+  - https://raw.githubusercontent.com/cloudify-cosmo/cloudify-manager/4.0.1/resources/rest-service/cloudify/types/types.yaml
+  - https://raw.githubusercontent.com/cloudify-cosmo/cloudify-fabric-plugin/1.5/plugin.yaml
+  - https://raw.githubusercontent.com/cloudify-incubator/cloudify-kubernetes-plugin/1.0.0/plugin.yaml
+  - common/docker.yaml
+  - common/networking.yaml
+
+inputs:
+  #kubernetes
+  kubernetes_master_ip:
```

Comment:
this should default to a secret, for example { get_secret: kubernetes_master_ip }.